codeforamerica/vita-min
# updates dev env credentials

## Link to pivotal/JIRA issue
n/a

## Is PM acceptance required? (delete one)
No - merge after code review approval

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Re-encrypted the dev env creds w/ a new encryption key
- Added the updated key to lastpass

## How to test?
- Pull the branch
- Copy the new encryption key from lastpass into `config/credentials/development.key`
- Ensure you can decrypt the file
